shared-components/tl-tournament-card.js

```diff
@@ -163,12 +194,14 @@ export class TournamentCard extends LitElement {
         // 2. To be replaced with live code
         //
         this._state = State.LOADING;
+        const data = await fetch(`../data/${entityId}.json`).then((res) =>
```

Comment:
Yes, we must consider all possible cases such as 401, 404, 500, timeout, etc.
I have to put them into `try catch`. 
However, in this project case, it will never happen as you know. so I didn't make it.
I can fix them if needed.